notifier/notifiers/http.go

```diff
@@ -0,0 +1,146 @@
+// Copyright 2015 clair authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package notifiers implements several kinds of notifier.Notifier
+package notifiers
+
+import (
+	"bytes"
+	"crypto/tls"
+	"crypto/x509"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"net/url"
+
+	"gopkg.in/yaml.v2"
+
+	"github.com/coreos/clair/config"
+	"github.com/coreos/clair/notifier"
+)
+
+// A HTTP notifier dispatches notifications to an HTTP endpoint.
+type HTTP struct {
```

Comment:
Can we rename this WebhookNotifier and s/http/webhook all over the place? I think that's more descriptive than the transport protocol (http).